files/ko/web/javascript/reference/global_objects/string/repeat/index.html

```diff
@@ -46,7 +46,7 @@ <h2 id="폴리필">폴리필</h2>
 
 <p><code>repeat</code>은 ECMAScript 2015 명세에 추가됐습니다. 따라서 어떤 표준 구현체에서는 사용할 수 없을 수도 있습니다. 그러나 아래 코드를 포함하면 지원하지 않는 플랫폼에서도 <code>repeat</code>을 사용할 수 있습니다.</p>
 
-<pre dir="rtl"><code>if (!String.prototype.repeat) {
+<pre><code>if (!String.prototype.repeat) {
```

Comment:
```suggestion
<pre class="brush: js"><code>if (!String.prototype.repeat) {
```
코드에 js 하이라이트 부분만 추가했습니다 :)